src/registrar/assets/js/get-gov.js

```diff
@@ -209,10 +206,31 @@ function handleInputValidation(e) {
 function handleValidationClick(e) {
   const attribute = e.target.getAttribute("validate-for") || "";
   if (!attribute.length) return;
+
   const input = document.getElementById(attribute);
   runValidators(input);
 }
 
+
+function handleFormsetValidationClick(e) {
+  // Check availability for alternative domains
+  
+  const alternativeDomainsAvailability = document.getElementById('check-avail-for-alt-domains');
+  
+  // Collect input IDs from the repeatable forms
+  let inputIds = Array.from(document.querySelectorAll('.repeatable-form input')).map(input => input.id);
+
+  // Run validators for each input
+  inputIds.forEach(inputId => {
+    const input = document.getElementById(inputId);
+    runValidators(input);
+  });
```

Comment:
Because `Array.from(document.querySelectorAll('.repeatable-form input'))` returns a list of inputs, couldn't you exclude the step where you are mapping out the input, then grabbing the element by the id?

For example (ignoring the weird spacing)...
```
let inputs =  Array.from(document.querySelectorAll('.repeatable-form input'))
inputs.forEach( input => {
   runValidators(input)
}
```